tutorials/plugins/editor/making_plugins.rst

```diff
@@ -367,21 +367,21 @@ The script could look like this:
     [Tool]
     public partial class CustomDock : EditorPlugin
     {
-        Control dock;
+        private Control _dock;
 
         public override void _EnterTree()
         {
-            dock = (Control)GD.Load<PackedScene>("addons/my_custom_dock/my_dock.tscn").Instantiate();
-            AddControlToDock(DockSlot.LeftUl, dock);
+            _dock = (Control)GD.Load<PackedScene>("res://addons/MyCustomDock/MyDock.tscn").Instantiate();
```

Comment:
nit: I think we prefer using the generic method instead, but I think it's fine either way.

```suggestion
            _dock = GD.Load<PackedScene>("res://addons/MyCustomDock/MyDock.tscn").Instantiate<Control>();
```

Reply:
Done!